src/components/accountTransactions/index.js

```diff
@@ -36,6 +37,7 @@ class accountTransactions extends React.Component {
 }
 
 const mapStateToProps = state => ({
+  delegateUsername: state.account.delegate ? state.account.delegate.username : null,
```

Comment:
This looks like you use the delegate name of the logged in account instead of the delegate name of the account of the active page.

Reply:
updated to grab it from the active account page